test/controllers/actions/AuthActionSpec.scala

```diff
@@ -283,28 +279,8 @@ class AuthActionSpec extends SpecBase with MockAppConfig with MockAuthConnector
 
               val result: Future[Result] = controller.onPageLoad()(fakeRequestWithMtditidAndNINO)
 
```

Comment:
Could do with adding a scenario where a user logs in with invalid credentials for secondary agent

`MockAuthConnector.authorise(authAction.secondaryAgentPredicate(mtditid.value))(Future.failed(InsufficientEnrolments()))`